github/REVIEWING.md

```diff
@@ -0,0 +1,341 @@
+# Git / GitHub  Reviewing Workflow
+
+Reviewing is done via the GitHub pull request interface, generally following the [GitHub guidelines](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests).
+
+Reviewing implies 5 actions:
+
+1. Reviewing the commit messages and metadata (signature, authorship)
+
+1. Reviewing the content of the contribution: good design, implementation value, readability, compliance
+
+1. Testing changes, making sure the fix the problem or they add the proposed feature, making sure they don't break anything
+
+1. Submitting the review
+
+1. Discussing and iterating
+
+All of these actions can rely on automation, such as linters, static analysis tools, unit testing / integration testing, typically included in CI/CD pipelines, such as those provided by [GitHub Actions](https://docs.github.com/en/actions).
+
+## Set Up
+
+Doing the review requires two items:
+
+1. Be logged in on GitHub and have reviewer access to the pull request.
+   The second part means that you have access to the pull request (in case of a private) repository.
+
+   In order to be formally assigned as a reviewer to a pull request in the GitHub interface, you have to have the [`Triage` role the repository](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization).
+   This role is assigned to you by a admin member of the repository.
+   This role is not a requirement, you can do a review without being assigned as a reviewer.
+
+1. Have a local clone of the repository (or a clone of your fork of the repository).
+   So you either did, at point, `git clone <URL>` using the `<URL>` of the main repository, or using the `<URL`> of your [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) of that repository.
+
+For the first item, you browser should have opened the GitHub pull request.
+
+For the second item, you will configure your local repository to point to the branch used to create the pull request.
+For this, do the following:
+
```

Comment:
I would mention `gh` here, since it saves a lot of time with the setup.
Not give instructions, just mention that everything in this step can be done via GitHub cli, and link to docs somewhere else (e.g. https://eszter.space/gh-cli/)

Reply:
Additionally, mention pressing <kbd>.</kbd> on the GitHub interface of the PR. It opens a cloud Code UI that has the branch of the PR already checked in so you don't even need to have your own clone locally.